examples/javascript-es5-browser/.eslintrc

```diff
@@ -3,6 +3,9 @@
     "adidas-es5",
     "adidas-env/browser"
   ],
+  "parserOptions": {
+    "ecmaVersion": 6
```

Comment:
Is it correct in `es5`?

Reply:
It looks weird, but else `eslint` complains that we cannot use modules without `es2015` or greater.

Reply:
OK.